www/src/utils/fetchGithub.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable @typescript-eslint/no-unsafe-argument, @typescript-eslint/no-unsafe-member-access, @typescript-eslint/no-unsafe-return, @typescript-eslint/no-unsafe-assignment */
```

Comment:
We could get rid of these disabled rules by
1. Casting the GitHub token as `string` (why is this not the default?)
2. Using Zod to validate the response. Is that worth it? cc @nexxeln @juliusmarminge

Reply:
casting is fine imo

Reply:
fixed em'